### DISCUSSION TITLE
Is this still working?

### DISCUSSION THREAD

**Discussion by romerocox** [netflie/whatsapp-cloud-api#219] [Q&A]
Hello everyone, I'm new here and I am trying to make sure this is still working and accurate. I will start reviewing the documentation and applying this to my projects if this is still working and being updated.

Thank you and I will make sure to support, if there is an option.

**Comment by cijagani** [comment]
Working perfectly.

**Comment by romerocox** [comment]
Yes!!! Will give it a try today, thanks!
